src/libs/ReportUtils.ts

```diff
@@ -865,7 +870,7 @@ function isArchivedRoom(report: OnyxEntry<Report> | EmptyObject): boolean {
 /**
  * Checks if the current user is allowed to comment on the given report.
  */
-function isAllowedToComment(report: Report): boolean {
+function isAllowedToComment(report: OnyxEntry<Report> | EmptyObject): boolean {
```

Comment:
```suggestion
function isAllowedToComment(report: OnyxEntry<Report>): boolean {
```

Is possible to avoid doing ` | EmptyObject`?

Reply:
I did a change in `Task.canModifyTask` take a look :)